src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -59,16 +64,51 @@ function teardown() {
         return
     fi
     kill_daemons $dir
-    if [ $(stat -f -c '%T' .) == "btrfs" ]; then
+    if [ `uname` != FreeBSD ] && \
+       [ $(stat -f -c '%T' .) == "btrfs" ]; then
         rm -fr $dir/*/*db
         __teardown_btrfs $dir
     fi
-    grep " $(pwd)/$dir/" < /proc/mounts | while read mounted rest ; do
+    grep " $(pwd)/$dir/" < ${PROCDIR}/proc/mounts | while read mounted rest ; do
         umount $mounted
     done
     rm -fr $dir
 }
 
+function timeout_diagnose() {
```

Comment:
@tchaikov 
Are you suggesting to take it out all the way?

Reason I left it in, is that I do run into these timeout problems that I woudl like to be diagnosed correctly.
Hence the full case statement. I'll add 124 explicitly, since that is what FreeBSD also does, if we decide to keep it.

Removing that from the tests, will result in somebody else concluding that this routine is not used.
And the submit a PR for removal.